From 29c36dcb9fea13d1b1c24227b0e4864e85ef7fe6 Mon Sep 17 00:00:00 2001
From: Aihua Xu <axu@cloudera.com>
Date: Wed, 16 Mar 2016 13:45:02 -0400
Subject: [PATCH 0535/1164] CDH-38433:
 TestCliDriver.testCliDriver_avro_decimal_old is
 failing on all Hive 5.x.x Jenkins jobs

Change-Id: I4cdebcb74f14316617758798e553182eee48f7f2
---
 .../results/clientpositive/avro_decimal_old.q.out  |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ql/src/test/results/clientpositive/avro_decimal_old.q.out b/ql/src/test/results/clientpositive/avro_decimal_old.q.out
index 22efe39..cb0b8f7 100644
--- a/ql/src/test/results/clientpositive/avro_decimal_old.q.out
+++ b/ql/src/test/results/clientpositive/avro_decimal_old.q.out
@@ -44,10 +44,10 @@ POSTHOOK: Input: default@avro_dec_old
 77.3
 55.7
 4.3
-6.0
+6
 12.3
 33.3
-19.0
+19
 3.2
 79.9
 PREHOOK: query: DROP TABLE avro_dec_old
-- 
1.7.9.5

